### ISSUE TITLE
Potential fix for false claim early in book

### ISSUE THREAD

**Issue by fernk0** [jirilebl/ca#1]
In chapter 1, section 2, it's stated that the exponential function given by $e^z$ is the unique continuous function on $\mathbb{C}$ that satisfies the functional equation $e^{z+w} = e^z e^w$ and $e^0 = 1$, but this is false considering $e^{az}$ is also such a function where $a$ is any fixed complex number.

Although it seems like this fact is never used, it could be amended to a true statement if, in addition, the condition $E(1) = e$ or even $\lim_{z\to 0} [(E(z) - 1)/z] = 1$ is added, where $E$ is a prospective functional solution to the previous conditions.

**Comment by jirilebl** [comment]
You're quite right.  I mixed up the initial condition with the initial condition one uses for the differential equation definition.  E(1)=e is the one I had in mind.  Thanks!

I usually credit errata on the errata page by name, what name should I put down?

**Comment by fernk0** [comment]
My name is Kevin Fernández, and also I have one last remark:

I actually don't know if even $E(1) = e$ might be enough by itself, since then $E(x + i y) = e^{x + 2 i y}$ would satisfy continuity, $E(0) = 1$, $E(1) = e$, and the sum-to-product relation. I think you might need the full strength of (complex-)differentiability at $z = 0$ with $E'(0) = 1$ to force conformality in some way. Of course, this last condition forces differentiability (and continuity and $E(0) = 1$) at each point as well, so it together with the sum-to-product relation should be enough but I haven't worked it out fully. There are many equivalent characterizations of the exponential for the real domain but I'm not too aware if these extend cleanly to the complex version.

**Comment by jirilebl** [comment]
Hmmm, you are right.

Note that $E(0)=1$ is unnecessary, that is automatic since $E(1+0)=E(1)E(0)$.

It is too late to think about this, so I will think about it more tomorrow :)  It is just an offhanded remark in the book, it is not needed.  It might even be reasonable to even just remove it entirely.  Or perhaps make add derivative condition.  I will think about it tommorrow when my mind is fresher :)

**Comment by jirilebl** [comment]
Just a quick thought before truly going to bed, $E(1)=e$ and $E(i)=e^i$ and might be enough since just integer powers of $e^i$ are dense in the unit circle.

**Comment by jirilebl** [comment]
I just took out the whole comment.  I don't really want to talk about the derivative at this point and it is not a terribly important remark.  I wonder if originally I put in the $e^0=1$ because it was the derivative at 0, but I didn't phrase it correctly or some such, it was a long time ago.  Sometimes trying to simplify some statement because you don't want to be too wordy just leads to saying wrong things, perhaps it's just best to not say anything :)

Anyway, thanks again for pointing it out.  It's good to shake out all these bugs.
